ledfx/effects/twod.py

```diff
@@ -74,6 +74,8 @@ def on_activate(self, pixel_count):
         # on change of effect
         self.t_height = self._virtual.config["rows"]
         self.t_width = self.pixel_count // self.t_height
+        # initialise here so inherited can assume it exists
+        self.current_time = timeit.default_timer()
```

Comment:
Moved the timer from from config_updated to on_activate

Reply:
Just make sure we come through on_activate, not sure I trust it...